synapse/lib/stormlib/infosec.py

```diff
@@ -0,0 +1,262 @@
+import math
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+import synapse.lib.node as s_node
+import synapse.lib.stormtypes as s_stormtypes
+
+CVSS31 = {
```

Comment:
It would be incredibly useful for future selfs if we documented where all of these magic numbers are coming from here.